test/auto/injectorSpec.js

```diff
@@ -302,6 +302,26 @@ describe('injector', function() {
           expect(instance.aVal()).toEqual('a-value');
         });
 
+        if (/chrome/.test(navigator.userAgent)) {
```

Comment:
Why wrap this in a browser sniff for Chrome only?  Since this test doesn't look browser specific and is just more about the injector doing the right thing, I think it'd make more sense to feature detect the support for ES6 classes

```js
  var supportsClasses = (function() {
    try {
      eval("class C {}");
      return true;
    } catch (err) {
      return false;
    }
  }());

  if (supportsClasses) { ... }
```